static/js/markdown.js

```diff
@@ -210,6 +210,15 @@ export function apply_markdown(message) {
                 match = match.replace(/>@/g, ">");
                 return match;
             });
+
+            // Silence quoted user group mentions.
+            const user_group_re = /<span.*user-group-mention.*data-user-group-id="(\d+)"[^>]*>@/gm;
```

Comment:
This regex seems likely wrong, in that you have a `.*` that could match a bunch of characters including another `><`, i.e. end up matching across multiple adjacent spans in a weird way

It appears the previous regex has the same bug; I think a `[^>]*` would work for the two `.*` blocks in the middle, so maybe do a commit fixing the existing instance and then fix this?

Reply:
Fixed the regex for this block and the previous block.